src/test/java/org/osiam/client/AbstractIntegrationTestBase.java

```diff
@@ -43,36 +45,29 @@
     protected static final String RESOURCE_ENDPOINT_ADDRESS = "http://localhost:8180/osiam-resource-server";
     protected static final String CLIENT_ID = "example-client";
     protected static final String CLIENT_SECRET = "secret";
-    protected OsiamConnector oConnector;
-    protected AccessToken accessToken;
+    protected static final OsiamConnector OSIAM_CONNECTOR;
+    protected static final Client CLIENT = ClientBuilder.newClient();
 
     static {
         OsiamConnector.setConnectTimeout(10000);
         OsiamConnector.setReadTimeout(30000);
-    }
 
-    @Before
-    public void abstractSetUp() throws Exception {
-        OsiamConnector.Builder oConBuilder = new OsiamConnector.Builder()
+        OSIAM_CONNECTOR = new OsiamConnector.Builder()
                 .setAuthServerEndpoint(AUTH_ENDPOINT_ADDRESS)
                 .setResourceServerEndpoint(RESOURCE_ENDPOINT_ADDRESS)
                 .setClientId(CLIENT_ID)
-                .setClientSecret(CLIENT_SECRET);
-        oConnector = oConBuilder.build();
-        accessToken = oConnector.retrieveAccessToken("marissa", "koala", Scope.ALL);
+                .setClientSecret(CLIENT_SECRET)
+                .setClientRedirectUri("http://localhost:5000/oauth2")
+                .build();
     }
 
-    protected void givenAnAccessTokenForOneSecond() {
-        OsiamConnector.Builder oConBuilder = new OsiamConnector.Builder()
-                .setAuthServerEndpoint(AUTH_ENDPOINT_ADDRESS)
-                .setResourceServerEndpoint(RESOURCE_ENDPOINT_ADDRESS)
-                .setClientId("short-living-client")
-                .setClientSecret("other-secret");
-        oConnector = oConBuilder.build();
-        accessToken = oConnector.retrieveAccessToken("marissa", "koala", Scope.ALL);
+    protected AccessToken accessToken;
+
+    protected void retrieveMarissaAccessToken() {
```

Comment:
What I mean is, that you created this method to retrieve the "vanilla" access token for `marissa` that's used almost everywhere in the tests. But I think there are still some places where you don't call this method, but actually retrieve the access token like `connector.retrieveAccessToken(...)`.